bin/replication.js

```diff
@@ -260,18 +273,28 @@ class _SetupReplication {
         const bucket = where === 'source' ? this._sourceBucket :
             this._targetBucket;
         this._s3Clients[where].createBucket({ Bucket: bucket }, (err, res) => {
-            if (err) {
+            if (err && err.code !== 'BucketAlreadyOwnedByYou') {
                 this._log.error('error creating a bucket', {
                     error: err.message,
+                    errCode: err.code,
```

Comment:
We could probably go ahead and log the entire error instead of just the error code.